pkg/cmd/admin/policy/modify_scc.go

```diff
@@ -0,0 +1,234 @@
+	SCCInterface kclient.SecurityContextConstraintsInterface
+
+	DefaultSubjectNamespace string
+	Subjects                []kapi.ObjectReference
+}
+
+func NewCmdAddSCCToGroup(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	options := &SCCModificationOptions{}
+
+	cmd := &cobra.Command{
+		Use:   name + " SCC GROUP [GROUP ...]",
+		Short: "Add groups to a security context constraint",
+		Long:  `Add groups to a security context constraint`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := options.CompleteGroups(f, args); err != nil {
+				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
+			}
+
+			if err := options.AddSCC(); err != nil {
+				kcmdutil.CheckErr(err)
+			}
+		},
+	}
+
+	return cmd
+}
+
+func NewCmdAddSCCToUser(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	options := &SCCModificationOptions{}
+	saNames := util.StringList{}
+
+	cmd := &cobra.Command{
+		Use:   name + " SCC USER [USER ...]",
+		Short: "Add users to a security context constraint",
+		Long:  `Add users to a security context constraint`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := options.CompleteUsers(f, args, saNames); err != nil {
+				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
+			}
+
+			if err := options.AddSCC(); err != nil {
+				kcmdutil.CheckErr(err)
+			}
+		},
+	}
+
+	cmd.Flags().VarP(&saNames, "serviceaccount", "z", "service account in the current namespace to use as a user")
+
+	return cmd
+}
+
+func NewCmdRemoveSCCFromGroup(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	options := &SCCModificationOptions{}
+
+	cmd := &cobra.Command{
+		Use:   name + " SCC GROUP [GROUP ...]",
+		Short: "Remove group from scc",
+		Long:  `Remove group from scc`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := options.CompleteGroups(f, args); err != nil {
+				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
+			}
+
+			if err := options.RemoveSCC(); err != nil {
+				kcmdutil.CheckErr(err)
+			}
+		},
+	}
+
+	return cmd
+}
+
+func NewCmdRemoveSCCFromUser(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	options := &SCCModificationOptions{}
+	saNames := util.StringList{}
+
+	cmd := &cobra.Command{
+		Use:   name + " SCC USER [USER ...]",
+		Short: "Remove user from scc",
+		Long:  `Remove user from scc`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := options.CompleteUsers(f, args, saNames); err != nil {
+				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
+			}
+
+			if err := options.RemoveSCC(); err != nil {
+				kcmdutil.CheckErr(err)
+			}
+		},
+	}
+
+	cmd.Flags().VarP(&saNames, "serviceaccount", "z", "service account in the current namespace to use as a user")
+
+	return cmd
+}
+
+func (o *SCCModificationOptions) CompleteUsers(f *clientcmd.Factory, args []string, saNames util.StringList) error {
+	if (len(args) < 2) && (len(saNames) == 0) {
```

Comment:
should this be an or?  Ie if I run the command just using service account names?